cfglib.py

```diff
@@ -179,11 +179,17 @@
     cfg.StrOpt('default_availability_zone', default="nova",
                help="Availability zone to use for VM provisioning, in case "
                     "source cloud zones do not match destination"),
-    cfg.StrOpt('ephemeral_copy_backend', default="rsync",
+    cfg.StrOpt('copy_backend', default="rsync",
```

Comment:
Can you please note that we using it to transport both ephemeral and cinder volumes in case of shared filesystem backends.